File: URI-py/1160.py
# -*- coding: utf-8 -*-
v = int(input())
for i in range (v):
    pa, pb, g1, g2 = map(float,input().split())
    a = 0
    while pa<=pb:
        pag1 = int((pa*(g1/100)))
        pbg2 = int((pb*(g2/100)))
        a += 1
        pa += pag1
        pb += pbg2
        if a > 100:
            break
    if a>100:
        print("Mais de 1 seculo.")
    else:
        print("%d anos."%a)

File: URI-cpp/1160.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;
    cin >> n;

    for (int i = 0; i < n; ++i) {
        int anos=0,pag1,pbg2,a=0,b=0;
        double ca=0,cb=0;

        cin >> a >> b >> ca >> cb;

        while (a<=b) {
            pag1 = a*(ca/100);
            pbg2 = b*(cb/100);
            a += pag1;
            b += pbg2;
            anos++;
            if (anos > 100) {
                break;
            }
        }
        if (anos>100) {
            printf("Mais de 1 seculo.\n");
        } else {
            printf("%d anos.\n",anos);
        }

    }

    return 0;
}